map/style_tests/style_symbols_consistency_test.cpp

```diff
@@ -1,31 +1,30 @@
-#include "testing/testing.hpp"
 #include "helpers.hpp"
+#include "testing/testing.hpp"
```

Comment:
As usual in our tests,
```
#include "testing/testing.hpp"
```
should be first, and in the case of long inclusion lists, it would be nice to separate it by a blank line from other include directives.